Sources/protoc-gen-swift/MessageGenerator.swift

```diff
@@ -167,7 +164,7 @@ class MessageGenerator {
     }
   }
 
-  func generateNested(printer p: inout CodePrinter, file: FileGenerator, parent: MessageGenerator?) {
+  func generateMainStuct(printer p: inout CodePrinter, file: FileGenerator, parent: MessageGenerator?) {
```

Comment:
Doh, I stared that that a few times saying something was wrong, but failed to notice it.  :(